spring-cloud-task-stream/pom.xml

```diff
@@ -42,7 +42,13 @@
 		</dependency>
 		<dependency>
 			<groupId>org.springframework.cloud</groupId>
-			<artifactId>spring-cloud-stream-test-support</artifactId>
+			<artifactId>spring-cloud-stream-test-support-internal</artifactId>
+			<version>1.0.0.BUILD-SNAPSHOT</version>
+			<scope>test</scope>
+		</dependency>
+		<dependency>
+			<groupId>org.springframework.cloud</groupId>
```

Comment:
When running the tests.  If Redis is up and running locally it works fine.  However if Redis is shutdown and tests are re-run, they fail with `Caused by: redis.clients.jedis.exceptions.JedisConnectionException: java.net.ConnectException: Connection refused`